bsp/esp32_p4_function_ev_board/esp32_p4_function_ev_board.c

```diff
@@ -315,16 +362,21 @@ esp_err_t bsp_touch_new(const bsp_touch_config_t *config, esp_lcd_touch_handle_t
     const esp_lcd_touch_config_t tp_cfg = {
         .x_max = BSP_LCD_H_RES,
         .y_max = BSP_LCD_V_RES,
-        .rst_gpio_num = GPIO_NUM_NC, // Shared with LCD reset
+        .rst_gpio_num = BSP_LCD_TOUCH_RST, // Shared with LCD reset
         .int_gpio_num = BSP_LCD_TOUCH_INT,
         .levels = {
             .reset = 0,
             .interrupt = 0,
         },
         .flags = {
             .swap_xy = 0,
+#if CONFIG_BSP_LCD_TYPE_1024_600
+            .mirror_x = 1,
+            .mirror_y = 1,
+#else
```

Comment:
It doesn't have to mirror the LCD because we usually position it vertically with the USB port facing upward to facilitate programming.  

What's more, it does not support mirroring through commands. You can see a warning message on the terminal. Instead, we should use IOs to control the 'UP/DN' and 'SHLR' signals in the LCD sub-board.

![image](https://github.com/user-attachments/assets/95d174d0-5d24-40d8-a7b6-c06bc1fc6e85)

Reply:
I know, that screen cannot use standard rotation, this is only for right rotation of the touch.

Reply:
> What's more, it does not support mirroring through commands. You can see a warning message on the terminal. Instead, we should use IOs to control the 'UP/DN' and 'SHLR' signals in the LCD sub-board.

We need any new API for `LVGL port` for rotation this screen type. I will think about it later.